src/cartridges/app_adyen_SFRA/cartridge/templates/default/account/payment/paymentForm.isml

```diff
@@ -45,10 +45,10 @@
             <!-- Cancel and Save Buttons -->
             <div class="row">
                 <div class="col">
-                    <a href="${URLUtils.url('PaymentInstruments-List')}" class="btn btn-block btn-outline-primary">${Resource.msg('button.cancel','account',null)}</a>
+                    <a href="${URLUtils.url('PaymentInstruments-List')}" class="btn btn-block btn-outline-primary" aria-label="Cancel and return to payment list">${Resource.msg('button.cancel','account',null)}</a>
```

Comment:
The text in aria-label should not be hard coded here so that it could be translated by merchant in the required locale